schema.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE animals (
     id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [x] <b>OPTIONAL:</b>   It would be nice to add `NOT NULL` and `PRIMARY KEY` constraints on ID just to make sure that it can't accept null values and it is primary key.